lib/beaker/test_suite.rb

```diff
@@ -172,6 +172,7 @@ def write_junit_xml(xml_file)
             end
             suite.add_child(properties)
 
+            @test_cases.sort! { |x,y| y.runtime <=> x.runtime } if time_sort
```

Comment:
good point.  I was doing this to make diff size smaller, but I'll change it soon.